.rubocop.yml

```diff
@@ -9,7 +9,8 @@ inherit_from:
 AllCops:
   DisplayCopNames: true
   DisplayStyleGuide: true
-  TargetRubyVersion: 2.6
+  TargetRubyVersion: 3.0
```

Comment:
🐘 I'm pretty sure we can leave this out, because [rubocop will look at our `.ruby-version`](https://docs.rubocop.org/rubocop/configuration.html#setting-the-target-ruby-version).